Lib/test/test_doctest/sample_doctest_skip.py

```diff
@@ -0,0 +1,37 @@
+"""This is a sample module used for testing doctest.
+
+This module includes various scenarios involving skips.
+"""
+
+# This test will pass.
+def no_skip():
+    """
+    >>> 2 + 2
+    4
+    """
+
+def single_skip():
+    """
+    >>> 2 + 2  # doctest: +SKIP
+    4
+    """
+
+def double_skip():
+    """
+    >>> 2 + 2  # doctest: +SKIP
+    4
+    >>> 3 + 3  # doctest: +SKIP
+    6
+    """
```

Comment:
```suggestion
def double_skip():
    """
    >>> 2 + 2  # doctest: +SKIP
    4
    >>> 3 + 3  # doctest: +SKIP
    6
    """
    
def one_skip_one_pass():
    """
    >>> 2 + 2  # doctest: +SKIP
    4
    >>> 3 + 3
    6
    """
```

Please, also add this corner case.

Reply:
I've added this to https://github.com/python/cpython/pull/117297.